modules/hcp-get-ocp-channel.adoc

```diff
@@ -50,7 +31,7 @@ version:
     - eus-4.16
     - fast-4.16
     - stable-4.16
-    image: quay.io/openshift-release-dev/ocp-release@sha256:b7517d13514c6308ae16c5fd8108133754eb922cd37403ed27c846c129e67a9a
+    image: quay.io/openshift-release-dev/ocp-release@sha256:xxxxx
```

Comment:
Are there placeholder conventions in openshift-docs?  `xxxxx` seems like a surprising choice.  I see you using `...` a few lines above; maybe you could use `sha256:...` here?

Reply:
`...` is reserved for the YAML file to show continuous code flow before and after. Using `<id>` placeholder instead of `xxxxx`